packages/snage/src/config/load.ts

```diff
@@ -5,9 +5,56 @@ import {parseConfig} from './parser';
 import {Config, Field, hasProvided, RawConfig, RawField} from './type';
 import * as E from 'fp-ts/lib/Either';
 import * as A from 'fp-ts/lib/Array';
+import * as O from 'fp-ts/lib/Option';
+import * as IO from 'fp-ts/lib/IO';
+import * as IOE from 'fp-ts/lib/IOEither';
 import {pipe} from 'fp-ts/lib/pipeable';
 import {getValueProvider} from '../provider/provider';
 import {getOrdering} from '../query/sort';
+import {ConfigParameterName, printAndExit} from '../command/common';
+import {flow} from 'fp-ts/lib/function';
+
+export const getConfig = (): E.Either<string, Config> => pipe(getConfigFile()(), E.chain(loadConfig));
+
+export const getConfigOrExit = (): Config => {
+    return pipe(
+        getConfig(),
+        E.getOrElse((err): Config => printAndExit(err)())
+    );
+};
+
+export const getConfigFile = (): IOE.IOEither<string, string> =>
+    pipe(
+        configPathFromArgs(),
+        O.alt(defaultConfigPath()),
+        IO.of,
+        IO.map(E.fromOption(() => `Could not find snage config. Use --${ConfigParameterName} to specify the config file.`))
+    );
```

Comment:
Hier ist mMn. eine Funktion zuviel, dadurch muss man die funktion 2x gecalled werden `getConfigFile()()`.
```suggestion
export const getConfigFile: IOE.IOEither<string, string> = () =>
    pipe(
        configPathFromArgs(),
        O.alt(defaultConfigPath()),
        E.fromOption(() => `Could not find snage config. Use --${ConfigParameterName} to specify the config file.`)
    );
```